operations/evaluate.go

```diff
@@ -28,12 +31,21 @@ func Evaluate() cli.Command {
 			cli.BoolFlag{
 				Name:  variantsFlagName,
 				Usage: "only show variant definitions",
-			}),
+			}, cli.BoolFlag{
+				Name:  useUpgradedYAMLFlagName,
+				Usage: "use upgraded YAML version",
+			}, cli.BoolFlag{
+				Name:  diffableFlagName,
+				Usage: "show the project configuration in an ordered, diff-friendly format",
+			},
+		),
 		Before: mergeBeforeFuncs(requirePathFlag),
 		Action: func(c *cli.Context) error {
 			path := c.String(pathFlagName)
 			showTasks := c.Bool(taskFlagName)
 			showVariants := c.Bool(variantsFlagName)
+			useUpgradedYAML := c.Bool(useUpgradedYAMLFlagName)
+			diffable := c.Bool(diffableFlagName)
```

Comment:
It seems like this is the preferred state, or at the very least, not a downgrade to the regular state. Why not make all the output diffable instead of making it flag dependent? Is it because it's substantially slower?

Reply:
I don't want to change the existing behavior of the function because it's a temporary feature that I don't really think is worth maintaining, and because people use `evergreen evaluate` in their scripts. Producing a different format would probably break those scripts.

Reply:
Wouldn't it only help scripts by giving the results in order?

Reply:
It would, but we would have to maintain the promise of sorted fields. I don't think we would really want to maintain that promise in the long-term unless someone actually asks for it.